wbia/dbio/ingest_database.py

```diff
@@ -1684,7 +1684,7 @@ def ingest_serengeti_mamal_cameratrap(species):
         >>> # SCRIPT
         >>> from wbia.dbio.ingest_database import *  # NOQA
         >>> import wbia
-        >>> species = ut.get_argval('--species', type_=str, default=wbia.const.TEST_SPECIES.ZEB_PLAIN)
+        >>> species = ut.get_argval('--species', type_=str, default=const.TEST_SPECIES.ZEB_PLAIN)
```

Comment:
missing import for `const`

Reply:
covered by `from wbia.dbio.ingest_database import *` which imports const. Can add redundant import for readability though